suites/portal/coremetadatapageTest.js

```diff
@@ -1,40 +1,30 @@
-// let nodes, nodes_list;
-//
-// Feature('Core Metadata');
-//
-// Scenario('test core metadata page', async(I) => {
-//
-//   // submit files
-//   await I.addNodes(I.getSheepdogRoot(), nodes_list);
-//   I.seeAllNodesAddSuccess(nodes_list);
-//
-//   I.goToFiles();
-//   I.loadFiles();
-//
-//   var fileNames = await I.grabTextFrom('//tr/td[2]');
-//   I.clickAFile(fileNames);
-//   I.loadCoreMetadata();
-//
-//   I.click('.boRitF'); // click back link
-//   I.loadFiles();
-//
-//   // delete files
-//   await I.deleteNodes(I.getSheepdogRoot(), nodes_list);
-//   I.seeAllNodesDeleteSuccess(nodes_list);
-//
-// });
-//
-// BeforeSuite(async (I) => {
-//   // try to clean up any leftover nodes
-//   await I.findDeleteAllNodes();
-// });
-//
-// Before((I) => {
-//   // load test data
-//   nodes = I.getNodePathToFile();
-//   nodes_list = I.sortNodes(Object.values(nodes));
-// });
-//
-// AfterSuite(async (I) => {
-//   await I.findDeleteAllNodes();
-// });
+// test data
+let metadata;
+
+Feature('CoreMetadataPageTest');
+
+const I = actor();
+
+BeforeSuite(async (nodes, sheepdog, pidgin, users) => {
+  await sheepdog.complete.findDeleteAllNodes();
+  await sheepdog.complete.addNodes(nodes.getPathToFile());
+  const validFile = nodes.getFileNode().clone();
+  await sheepdog.complete.addNode(validFile);
+  metadata = await pidgin.do.getCoremetadata(validFile, 'application/json', users.mainAcct.accessTokenHeader);
```

Comment:
i know this line depends on the backend not the frontend, but i still think it should be in the test, not in BeforeSuite. and it should include `pidgin.ask.seeJsonCoremetadata(valid_file, metadata);` too

When we see the tests failed during "BeforeSuite" the initial reaction is to think it's not an actual failing test